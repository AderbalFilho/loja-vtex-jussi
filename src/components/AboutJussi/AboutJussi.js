import { JButton, JSection, JText, JTitle } from '@ui';

import imageJussi1x from '@assets/img/image-jussi.png';
import imageJussi2x from '@assets/img/image-jussi@2x.png';

import './AboutJussi.scss';

function AboutJussi() {
  return (
    <JSection classes="about-jussi">
      <div className="about-jussi__container">
        <div className="about-jussi__text-container">
          <JTitle tag="h2" classes="about-jussi__title">
            Olá, somos a Jüssi
          </JTitle>
          <JText color="black" classes="about-jussi__description">
            A Jüssi é uma agência integrante do grupo global WPP que vem há 10
            anos consolidando o pensamento voltado para produtos e resolução de
            problemas. Nosso área dedicada exclusivamente para Produtos Digitais
            é organizada em 6 especialidades: Product Managamenet, User
            Experience Design, SEO, Tecnologia, Agile e User Behavior Analytics.
          </JText>
          <JButton
            tag="a"
            href="https://jussi.com.br/"
            target="_blank"
            rel="noreferrer noopener"
            classes="about-jussi__button"
          >
            Conheça a Jüssi
          </JButton>
        </div>
        <img
          src={imageJussi1x}
          srcSet={`${imageJussi1x}, ${imageJussi2x} 2x`}
          alt="Escritório da Jüssi, mostrando os funcionários trabalhando em suas mesas"
          className="about-jussi__img"
        />
      </div>
    </JSection>
  );
}

export default AboutJussi;
