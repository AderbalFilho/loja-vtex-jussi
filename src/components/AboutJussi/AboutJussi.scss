.about-jussi {
  padding: 64px 0;
}

.about-jussi__container {
  display: flex;
  margin: 0 auto;
  justify-content: space-between;
  max-width: 1024px;
  width: 100%;
}

.about-jussi__text-container {
  width: 241px;
}

.about-jussi__title {
  margin: 0;
}

.about-jussi__description {
  margin: 24px 0;
}

.about-jussi__button {
  display: inline-block;
}

.about-jussi__img {
  max-height: 520px;
  max-width: 676px;
}
