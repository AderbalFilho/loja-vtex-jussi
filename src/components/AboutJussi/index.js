import AboutJussi from './AboutJussi';

export default AboutJussi;
