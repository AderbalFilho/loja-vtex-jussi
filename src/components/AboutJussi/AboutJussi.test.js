import React from 'react';
import TestRenderer from 'react-test-renderer';

import AboutJussi from './AboutJussi';

describe('AboutJussi', () => {
  it('matches snapshot', () => {
    const testRenderer = TestRenderer.create(<AboutJussi />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
