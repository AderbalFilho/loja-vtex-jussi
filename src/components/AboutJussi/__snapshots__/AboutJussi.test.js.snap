// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutJussi matches snapshot 1`] = `
<section
  className="j-section j-section--white about-jussi"
>
  <div
    className="about-jussi__container"
  >
    <div
      className="about-jussi__text-container"
    >
      <h2
        className="j-title j-title--black j-title--large about-jussi__title"
      >
        Olá, somos a Jüssi
      </h2>
      <p
        className="j-text j-text--black j-text--normal about-jussi__description"
      >
        A Jüssi é uma agência integrante do grupo global WPP que vem há 10 anos consolidando o pensamento voltado para produtos e resolução de problemas. Nosso área dedicada exclusivamente para Produtos Digitais é organizada em 6 especialidades: Product Managamenet, User Experience Design, SEO, Tecnologia, Agile e User Behavior Analytics.
      </p>
      <a
        className="j-button j-button--back-transparent j-button--border-black j-button--black j-button--rad-normal j-button--link about-jussi__button"
        href="https://jussi.com.br/"
        rel="noreferrer noopener"
        target="_blank"
      >
        Conheça a Jüssi
      </a>
    </div>
    <img
      alt="Escritório da Jüssi, mostrando os funcionários trabalhando em suas mesas"
      className="about-jussi__img"
      src=""
      srcSet=",  2x"
    />
  </div>
</section>
`;
